backend/src/main/kotlin/be/osoc/team1/backend/entities/Project.kt

```diff
@@ -32,7 +32,7 @@ class Position(
     val amount: Int,
     @JsonIgnore
     @NotBlank
-    val edition: String = ""
+    var edition: String = ""
```

Comment:
Why did you change this?

Reply:
This was changed because there is no other way to patch a project and have the edition fields be set on the positions without requiring the frontend to provide them. You can't make new positions because that would give them new id's so you have to change the existing edition field.